NOTICE.txt

```diff
@@ -8,6 +8,48 @@ This document includes a list of open source components used in Mattermost Web A
 
 --------
 
+## @formatjs/intl-pluralrules
+
+This product contains '@formatjs/intl-pluralrules' by FormatJS.
+
+Polyfill for Intl.PluralRules
```

Comment:
Not sure if the description is correct.

Reply:
Yeah, that description is correct. `Intl.PluralRules` is something that the browser should provide, but some browsers don't support it yet.